services/galley/src/Galley/API/Query.hs

```diff
@@ -89,6 +89,7 @@ getConversations zusr range size =
       | Data.isConvDeleted c = Data.deleteConversation (Data.convId c) >> pure False
       | otherwise = pure True
 
+-- TODO(mheinzel): in minimal API, but not essential
```

Comment:
so, it should be removed from minimal api?  i'm not sure that's what this says.  maybe i don't have enough context to review this, in that case please disregard.

Reply:
I removed the comment in fd86f75, was just a note to myself saying that this is part of the "minimal API" [1], so we will want to do it soonish, but it's not an endpoint we need to get a basic federation integration test working.

[1] the API implemented in AVES (https://aves.services.zinfra.io/swagger#!/), which is a good reference point, as it pretty much corresponds to what you need to make our clients work for basic use cases (but without teams).